corehq/apps/translations/management/commands/translate_po_files.py

```
@@ -0,0 +1,672 @@
+        def wrapper(*args, **kwargs):
+            num_retries = 0
+            delay = initial_delay
+
+            while True:
+                try:
+                    return func(*args, **kwargs)
+                except errors:
+                    num_retries += 1
+                    print("Rate limit error, retrying with backup model: gpt-4o")
+                    return func(*args, **kwargs, backup_model='gpt-4o')
+                except errors:
```

Ruff (0.8.2)

try-except block with duplicate exception `errors`

(B025)